service/collector_test.go

```diff
@@ -135,6 +135,53 @@ func TestCollector_Start(t *testing.T) {
 	}, time.Second*2, time.Millisecond*200)
 }
 
+// Calling shutdown before collector is running should not stop it from running
```

Comment:
nit: The comment here should start with the func name, alternatively the comment could be moved inside func